x-pack/test/security_solution_api_integration/test_suites/detections_response/detection_engine/rule_execution_logic/trial_license_complete_tier/execution_logic/eql.ts

```diff
@@ -746,5 +746,140 @@ export default ({ getService }: FtrProviderContext) => {
         expect(fullAlert?.['host.asset.criticality']).to.eql('high_impact');
       });
     });
+
+    describe('using data with a @timestamp field', () => {
+      it('specifying only timestamp_field results in a warning, and no alerts are generated', async () => {
+        const rule: EqlRuleCreateProps = {
+          ...getEqlRuleForAlertTesting(['auditbeat-*']),
+          timestamp_field: 'event.created',
+        };
+
+        const {
+          previewId,
+          logs: [_log],
+        } = await previewRule({ supertest, rule });
+
+        expect(_log.errors).to.be.empty();
+        expect(_log.warnings).to.eql([
+          'This rule reached the maximum alert limit for the rule execution. Some alerts were not created.',
+        ]);
+
+        const previewAlerts = await getPreviewAlerts({ es, previewId });
+        expect(previewAlerts.length).to.be.greaterThan(0);
+      });
+
+      it('specifying only timestamp_override results in an error, and no alerts are generated', async () => {
+        const rule: EqlRuleCreateProps = {
+          ...getEqlRuleForAlertTesting(['auditbeat-*']),
+          timestamp_override: 'event.created',
+        };
+
+        const {
+          previewId,
+          logs: [_log],
+        } = await previewRule({ supertest, rule });
+
+        expect(_log.errors).to.be.empty();
```

Comment:
test title suggests no alerts were generated and error encountered
no error in test and alert were generated
`specifying only timestamp_override results in an error, and no alerts are generated`

Reply:
f3e55d52877